src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add `XmlEncoder::SAVE_OPTIONS` context option
+ * Add `BackedEnumNormalizer::NULL_IF_NOT_PART_OF_ENUMERATION` context option
```

Comment:
```suggestion
 * Add `BackedEnumNormalizer::ALLOW_INVALID_VALUE` context option
```
Or `NULL_IF_NOT_PART_OF_ENUMERATION`, depending on which one you want to keep

Reply:
yes sorry, I've just changed my mind on the naming afterwards